src/model/guild/emoji.rs

```diff
@@ -223,7 +223,11 @@ impl Display for Emoji {
     ///
     /// This is in the format of: `<:NAME:EMOJI_ID>`.
```

Comment:
This should mention the format for animated emojis.
```suggestion
    /// This is in the format of either `<:NAME:EMOJI_ID>` for normal emojis, 
    /// or `<a:NAME:EMOJI_ID>` for animated emojis.
```